src/DotNetLightning.Core/Channel/ChannelError.fs

```diff
@@ -368,9 +369,17 @@ module internal OpenChannelMsgValidation =
         else
             Ok()
 
-    let checkFundingSatoshisLessThanMax (msg: OpenChannelMsg) =
-        if (msg.FundingSatoshis >= ChannelConstants.MAX_FUNDING_SATOSHIS) then
-            sprintf "funding_satoshis must be less than %A. It was %A" ChannelConstants.MAX_FUNDING_SATOSHIS msg.FundingSatoshis
+    let checkFundingSatoshisLessThanMax (msg: OpenChannelMsg) (localParams: LocalParams) (isOurOpenChannelMsg: bool) =
+        // If we are validating our own open message we make sure we are forcing support for option_support_large_channel on the other peer
+        let featureType =
+            match isOurOpenChannelMsg with
+            | true ->
+                Some FeaturesSupport.Mandatory
+            | false ->
+                None
+
+        if (msg.FundingSatoshis >= ChannelConstants.MAX_FUNDING_SATOSHIS && not (localParams.Features.HasFeature (Feature.OptionSupportLargeChannel, ?featureType = featureType))) then
```

Comment:
1. split this line, it's too long
2. now that you're changing it, let's remove the unneeded parentheses (C# needs them, F# doesn't)